catlearn/relation_cache.py

```diff
@@ -193,7 +194,7 @@ def __init__(
         self._graph = CompositionGraph[NodeType, ArrowType, Tsor](
             self._graph_comp(), ())
 
-        # register wraping attributes
+        # register wraping attributestriplets = 
```

Comment:
What does this comment do? It looks like a pragma for an out of band analyser/checker

Reply:
> What does this comment do? It looks like a pragma for an out of band analyser/checker

Or the IDE jumping somewhere random and I didn't see I'd added stuff to the comment because of this :p

Reply:
x)